client/src/redux/actions.ts

```diff
@@ -0,0 +1,43 @@
+import { getFromStorage, setInStorage } from 'API/localStorage';
+import { mockFeedbackData } from 'components/Data/mockFeedbackData';
+import { FEEDBACK_LOCALSTORAGE_LABEL } from 'constants/labels';
+import {
+  FEEDBACK_REQUEST_FAILED,
+  FEEDBACK_REQUEST_START,
+  FEEDBACK_REQUEST_SUCCESS
+} from 'redux/feedbackReducer';
+import { Feedback } from 'types';
+
+export const getFeedback = () => {
+  return dispatch => {
+    try {
+      dispatch({ type: FEEDBACK_REQUEST_START });
+      const feedbackData = getFromStorage(FEEDBACK_LOCALSTORAGE_LABEL);
+      dispatch({
+        type: FEEDBACK_REQUEST_SUCCESS,
+        payload: { data: feedbackData || mockFeedbackData }
+      });
+      if (!feedbackData) setInStorage(FEEDBACK_LOCALSTORAGE_LABEL, mockFeedbackData);
+    } catch (error) {
+      dispatch({ type: FEEDBACK_REQUEST_FAILED, payload: error });
+    }
+  };
+};
+
+export const addFeedback = (newFeedback: Feedback) => {
+  return dispatch => {
+    try {
+      const feedback = getFromStorage(FEEDBACK_LOCALSTORAGE_LABEL);
+      if (feedback) {
+        const updatedFeedbackData = [newFeedback, ...feedback];
+        setInStorage(FEEDBACK_LOCALSTORAGE_LABEL, updatedFeedbackData);
+        dispatch({ type: FEEDBACK_REQUEST_SUCCESS, payload: { data: updatedFeedbackData } });
+      } else {
+        setInStorage(FEEDBACK_LOCALSTORAGE_LABEL, [newFeedback]);
+        dispatch({ type: FEEDBACK_REQUEST_SUCCESS, payload: { data: [newFeedback] } });
+      }
```

Comment:
this part can be refactored, because in both cases you need to call `setInStorage` and `dispatch` functions with almost the same params. please rewrite it with no use if/else statement (ternary operator can be used)